i18n/zh/docusaurus-plugin-content-docs/current/user-manuals/advancedstatefulset.md

```diff
@@ -364,6 +437,49 @@ spec:
       paused: true
 ```
 
+### PersistentVolumeClaim 更新策略
+
+**FEATURE STATE:** Kruise v1.8.0
+
+如果你在安装或升级 Kruise 的时候启用了 `StatefulSetAutoResizePVCGate` feature-gate， 你可以使用
+`.spec.volumeClaimUpdateStrategy` 字段来控制 PVC 的更新策略。
+
+```yaml
+apiVersion: apps.kruise.io/v1beta1
+kind: StatefulSet
+spec:
+  # ...
+  volumeClaimUpdateStrategy:
+    # 可选值有 OnPodRollingUpdate 和 OnDelete
+    #   OnPodRollingUpdate ：controller 会在 pod 升级过程中自动扩容 pvc
+    #   OnDelete ：controller 只会在 pvc 删除后使用新的 volume claim template 重建 pvc
+    #             即 kruise 1.7 版本之前默认的 pvc 更新策略。
+    type: OnPodRollingUpdate 
+```
+
+#### 校验规则（OnPodRollingUpdate 模式）
+
+当启用 `OnPodRollingUpdate` 时，Kruise Webhook 会对`.spec.volumeClaimTemplates` 变更进行强制校验：
+
+1. **禁止 PVC 数量增减**：不可修改 `.spec.volumeClaimTemplates` 的 PVC 数量
+2. **扩容限制**： 只允许对支持扩展的 StorageClass 进行 PVC 容量扩容
+3. **缩容（扩容后回滚）限制**：
+   - 需遵循扩容限制
+   - 允许缩小对应卷模版的 `.spec.resources.requests.storage` 字段值
+   - **不会对现有 PVC 执行实际缩容操作**（仅更新模板配置）
+
+#### 注意事项
+
+- StorageClass 需支持动态扩展才能生效 PVC 扩容操作
+- 修改 PVC 模板前请确保集群已启用对应 FeatureGate
+- OnDelete 策略下需手动删除 PVC 才能触发新模板 PVC 的应用
+
+### Partition 语义说明
```

Comment:
we can reference the [cloneset docs](https://openkruise.io/docs/user-manuals/cloneset#partition) to add a partition section in "update features"